### ISSUE TITLE
User testing

### ISSUE THREAD

**Issue by jsms90** [CYPIAPT-LNDSE/anxiety-ladder#86]
I apologise in advance for the length. But I sincerely hope this helps.

Here are the links that I shared with you about how to conduct user testing, just so that we have them here on github for future reference:
https://library.gv.com/the-gv-research-sprint-finalize-schedule-and-complete-interview-guide-day-3-b8cddb8f931d#.82x16birz
https://www.youtube.com/watch?v=U9ZG19XTbd4

I am aware that there is some tension between the need to test the app in its current prototype form, and the desire to present the entire idea to users, in order to validate the wider concept and be able to present _that_ to the Anna Freud Centre. So I suggest that you split each of your "interviews" with your users into two sections:

1. **User testing** - focusing on the usability of the _current_ app
  + It is important to highlight that you need their help in testing _the app_, but you are not testing _them_ in any way
  + You might want to ask some context-setting questions
    + Have they heard of any apps that help young people to track their anxiety?
    + Have they ever used one before? (for themselves, or with a yp - if it's a clinician)
    + If so, what did they like, what did they not like? What was helpful, what was not?
  + Introduce the prototype. (Make sure you are only trialling this on a mobile, whether it is yours or their own) and tell them the user journey that you want them to perform i.e. adding a new goal/ladder - whichever terminology you would use with a yp/clinician. Let them know in advance, where you want them to stop i.e. once they have finished their ladder
    + This is important because you shouldn't be explaining to them what you want them to do when they are on each page, or asking them to click on anything in particular (e.g. the cloud with the +), or _not_ to click on anything (e.g. the clouds which are designed to represent goals that they have already entered into the app). The point is to set up the situation, and see how intuitive the experience is for them.
    + If you want, you could specifically set them the task of filling in the app as if their goal were: "eating lunch in the canteen" (to reduce any confusion about entering a different goal, and then seeing the placeholder text on the 3rd page). We want to reduce any confusion about this being a prototype, rather than a fully fledged version of the product, so that we get feedback on things that may need to change.
  + Ask them to think aloud!
    + The most valuable information you can glean from them is their own, organically developing thoughts, with as little influence from you as possible as to what the "right" or desired answer might be
    + Try to ask questions that probe for their own response. If you see them look confused or surprised, but they don't say anything, try asking "What happened there?" rather than indicating what you think they might have been reacting to or why
  + Ask some summary questions at the end. Some good things to include might be:
    + How do you feel, using the app?
    + What were the best / worst parts?
    + Would you prefer using this app, or using --insert current method (paper forms?)--
    + Is there anything missing from the app which you would like to see / would have expected to see?

2. **User research** - finding out which feature(s) they would like us to build next
  + This is the part where we find out _what they need most_ from us, not what we want to build for them. Since we have some pretty well formed ideas about where we see this app going, this part can be difficult, but try to formulate some questions that allow them creative freedom to tell you what they need
    + These questions are probably easier for you to draft than for me, as you know the material that is provided to young people.
    + Try starting with some relatively open ended questions, to ascertain what struggles they are facing. These questions are the ones that might be different, depending on whether you are talking to a yp or a clinician.
  + At the end of this questioning, once you have had a chance to find things out from their perspective, you could introduce our ideas for further features e.g. filling out more sections when they are starting an exposure, when they have finished (reviewing) & visualisations of their anxiety rating history

**Comment by ishiar** [comment]
@jsms90 jen that's amazing help! It is quite abstract what could help you, your text above really helped to understand what you need, so no apologies at all... think that is really biggest lesson learnt for me during the last couple of days that you just need to talkntalkntalk and say it over again and again, as you guys know what I don't and vice versa. so really highly appreciated the above. 
I started to show the app yesterday and I do find it quite challenging to truly open up for feedback - these cbt psych-interventions are so super pre-defined, which means it's really hard to open up a conversation about 'what should come next' but I try my best :D . one thing, I'm absolutely fine with it if you are ok with it - once you save a new goal, the app sends you automatically into the canteen goal to insert your challenges - which makes no sense but I can understand that there was not enough time for the prototype - but folks are completely lost after that - would you let them be confused or explains what is happening there?..
triple thanks for your really amazing amount of energy!!! do my best!

**Comment by jsms90** [comment]
@ishiar no, that makes perfect sense. That's what I was trying to avoid with:
> If you want, you could specifically set them the task of filling in the app as if their goal were: "eating lunch in the canteen" (to reduce any confusion about entering a different goal, and then seeing the placeholder text on the 3rd page).

Until this is able to go beyond a mere prototype, and we can pass data between pages, rather than having a pre-defined click through experience, the best way you can conduct the user testing is by heading off that confusion in advance. Let them know what goal you want them to put into the app :+1:

**Comment by ishiar** [comment]
@jsms90 aaaaahh get it!!!!
